tests/framework/Models/External/tests

```diff
@@ -20,6 +20,7 @@
   output = 'GrayModelOutput.txt'
   python3_only = True
   minimum_library_versions = 'pyomo 6.4'
+  required_libraries = 'cmake glpk ipopt cyipopt'
```

Comment:
required_libraries uses similar logic to the library check that causes skip_check='True' to be needed, so this will never be called.

Reply:
Otherway round, skip_check basically tries "import cmake" which will not work.  Assuming that "import ipopt" works, changing required_libraries to ipopt is good.

Reply:
Removed skip check for ipopt only. The required library is now 'ipopt' only.